webapp/src/main/scala/org/allenai/common/webapp/InfoRoute.scala

```diff
@@ -37,14 +39,14 @@ class InfoRoute(val info: Map[String, String] = Map.empty) {
       pathEndOrSingleSlash {
         respondWithMediaType(MediaTypes.`application/json`) {
           complete {
-            info.toJson.prettyPrint
+            infoWithTime.toJson.prettyPrint
```

Comment:
I don't like that because then you have to call it explicitly when you use it. But I don't dislike it very strongly, so I made the change anyways.